backend/create_test_storage_dir.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+# Create all test_storage_dir subtree needed for running test
+TEST_STORAGE_DIR_NAME="test_storage_dir"
+ORIGIN="."
+
+mkdir $ORIGIN/$TEST_STORAGE_DIR_NAME
+mkdir $ORIGIN/$TEST_STORAGE_DIR_NAME/depot
+mkdir $ORIGIN/$TEST_STORAGE_DIR_NAME/previews
+mkdir $ORIGIN/$TEST_STORAGE_DIR_NAME/radicale_storage
+mkdir $ORIGIN/$TEST_STORAGE_DIR_NAME/sessions
+mkdir $ORIGIN/$TEST_STORAGE_DIR_NAME/sessions/sessions_data
+mkdir $ORIGIN/$TEST_STORAGE_DIR_NAME/sessions/sessions_lock
```

Comment:
Need some logging (like we have in setup_default_backend.sh)

Reply:
done.